ivy/data_classes/container/experimental/utility.py

```diff
@@ -1,5 +1,82 @@
+# global
+from typing import Optional, Union, Dict, List
+
+# local
+import ivy
 from ivy.data_classes.container.base import ContainerBase
 
 
 class _ContainerWithUtilityExperimental(ContainerBase):
-    pass
+    @staticmethod
+    def static_optional_get_element(
+        x: Optional[Union[ivy.Array, ivy.Container]] = None,
+        /,
+        *,
+        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+        to_apply: bool = True,
+        prune_unapplied: bool = False,
+        map_sequences: bool = False,
+        out: Optional[ivy.Container] = None,
+    ) -> ivy.Container:
+        """
+        ivy.Container static method variant of ivy.optional_get_element. This method
+        simply wraps the function, and so the docstring for ivy.optional_get_element
+        also applies to this method with minimal changes.
+
+        Parameters
+        ----------
+        x
+            container with array inputs.
+        key_chains
+            The keychains to apply or not apply the method to. Default is ``None``.
+        to_apply
+            If True, the method will be applied to key_chains, otherwise key_chains
+            will be skipped. Default is ``True``.
+        prune_unapplied
+            Whether to prune key_chains for which the function was not applied.
+            Default is ``False``.
+        map_sequences
+            Whether to also map method to sequences (lists, tuples).
+            Default is ``False``.
+        out
+            optional output container, for writing the result to.
+
+        Returns
+        -------
+        ret
+            Container with arrays flattened at leaves.
+        """
+        return ContainerBase.cont_multi_map_in_function(
+            "optional_get_element",
+            x,
+            key_chains=key_chains,
+            to_apply=to_apply,
+            prune_unapplied=prune_unapplied,
+            map_sequences=map_sequences,
+            out=out,
+        )
+
+    def optional_get_element(
+        self: ivy.Container,
+        /,
+        *,
+        out: Optional[ivy.Container] = None,
+    ) -> ivy.Container:
+        """
+        ivy.Container instance method variant of ivy.optional_get_element. This method
+        simply wraps the function, and so the docstring for ivy.optional_get_element
+        also applies to this method with minimal changes.
+
+        Parameters
+        ----------
+        self
+            Input container
+        out
+            Optional output container, for writing the result to.
+
+        Returns
+        -------
+        ret
+            Output container.
+        """
+        return self.static_optional_get_element(self, out=out)
```

Comment:
Not sure if this was here before, but what are these `optional_get` changes? It's not related to `crop` is it?

Reply:
Your main branch is not synced with ours.
![image](https://github.com/unifyai/ivy/assets/111577222/c1192715-f438-4bda-b8dc-855140be9555)

Reply:
and you also seem to have pushed changes to the `test_array_api` files. We only merge PRs that make changes related to the task they are resolving. Anything extra needs to be removed.

Reply:
I have synced my main branch although I am unable to access and reset or revert test_array_api as it is a subfolder containing 31 files that I have not touched. Could this problem be due to other users participating in my pull-request? Sorry for all the trouble, but all of these problems besides the lint errors have not been due to changes made by me so I am very confused as to why they are happening.

Reply:
I would suggest making a new branch for each task you start working on. This will make it easy to resolve such issues in the future. 
I think the most straightforward way to deal with this now is to 1) make a new fork, 2) create a branch and copy-paste your function and test there, 3) make a new PR from this branch.